Formula/enscript.rb

```diff
@@ -4,6 +4,7 @@ class Enscript < Formula
   url "https://ftp.gnu.org/gnu/enscript/enscript-1.6.6.tar.gz"
   mirror "https://ftpmirror.gnu.org/enscript/enscript-1.6.6.tar.gz"
   sha256 "6d56bada6934d055b34b6c90399aa85975e66457ac5bf513427ae7fc77f5c0bb"
+  license "GPL-3.0-or-later"
```

Comment:
https://git.savannah.gnu.org/gitweb/?p=enscript.git;a=blob;f=README;h=d2f1c4a8a3e6ed8b60615ac6201eca5c58420e25;hb=0b6eabf156d0b83110b521e589d67b0d98666be0

> Enscript is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.